eol.d.ts

```diff
@@ -23,6 +23,13 @@ declare module eol {
    */
   export function cr(text: string): string;
 
+  /**
+   * Dubnormalize. Used internally. Mixin friendly.
+   * Create normalizer function from replacement text
+   * @return anonymous function
+   */
+  export function dub(text: string): function;
```

Comment:
@FiniteLooper This javascript `dub` method returns an anonymous function. What typescript return type would you advise? `function`? `object`? `any`??

Reply:
Seems correct now but always open to opinion @FiniteLooper